app/javascript/components/common/MentorDiscussionSummary.tsx

```diff
@@ -30,7 +30,7 @@ export const MentorDiscussionSummary = ({
         />
         {postsCount}
       </div>
-      <time dateTime={createdAt}>{fromNow(createdAt)}</time>
+      <time dateTime={createdAt}>{fromNow(createdAt, true)}</time>
```

Comment:
I found that the boolean nature of the argument made it hard to understand what its function is. Maybe introduce a separate function: `shortFromNow` or something like that?

Reply:
Done. Thanks.